index.html

```diff
@@ -195,13 +195,12 @@ <h2>Attributes</h2>
               <p>Let <var>delay</var> be the argument to the setter.</p>
             </li>
             <li>
-              <p>If <var>delay</var> is negative, <a>throw</a> an
-              <code>TypeError</code> and abort these steps.</p>
+              <p>If <var>delay</var> is negative or larger than 4 seconds then,
```

Comment:
@foolip You changed InvalidAccessError to TypeError. However, doesn't RangeError make more sense?

Reply:
Yeah, that'd work too.

Reply:
Change it to RangeError.